packages/system/src/icons/icon.css.ts

```diff
@@ -14,9 +14,9 @@ export const iconStyle = recipe({
 				width: pxToRem(24),
 			},
 			full: {
-				height: '100%',
-				width: '100%',
-			}
+				height: "100%",
+				width: "100%",
+			},
```

Comment:
I'm surprised there isn't a rule in the `.prettierrc` about this.

Reply:
True! Want me to add `"singleQuote": true`? Might as well do it in another PR as it will change a lot of files

Reply:
Sure, sounds like a good idea to me! Go ahead and add an issue for it first. Nothing like that wonderful feeling of closing an issue with a PR...